packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_submit_dfc_htlc.test.ts

```diff
@@ -0,0 +1,177 @@
+import { MasterNodeRegTestContainer, GenesisKeys } from '@defichain/testcontainers'
+import { getProviders, MockProviders } from '../provider.mock'
+import { P2WPKHTransactionBuilder } from '../../src'
+import { calculateTxid, sendTransaction } from '../test.utils'
+import { WIF } from '@defichain/jellyfish-crypto'
+import BigNumber from 'bignumber.js'
+import { OP_CODES } from '@defichain/jellyfish-transaction'
+import { ICXSubmitDFCHTLC } from '@defichain/jellyfish-transaction/script/dftx/dftx_icxorderbook'
+import { ICXDFCHTLCInfo, ICXListHTLCOptions } from '@defichain/jellyfish-api-core/category/icxorderbook'
+import { Testing } from '@defichain/jellyfish-testing'
+import { icxorderbook } from '@defichain/jellyfish-api-core'
+
+describe('submit DFC HTLC', () => {
+  const container = new MasterNodeRegTestContainer()
+  const testing = Testing.create(container)
+  let providers: MockProviders
+  let builder: P2WPKHTransactionBuilder
+
+  beforeAll(async () => {
+    await container.start()
```

Comment:
You can use testing container from now on

Reply:
testing.container.start()